yt_dlp/postprocessor/ffmpeg.py

```diff
@@ -659,12 +660,32 @@ def run(self, info):
 
         temp_filename = prepend_extension(filename, 'temp')
         self.to_screen('Embedding subtitles in "%s"' % filename)
-        self.run_ffmpeg_multiple_files(input_files, temp_filename, opts)
-        os.replace(temp_filename, filename)
+        if info['ext'] in self.AUDIO_EXTS:
+            self.embed_lyrics(input_files)
+        else:
+            self.run_ffmpeg_multiple_files(input_files, temp_filename, opts)
+            os.replace(temp_filename, filename)
 
         files_to_delete = [] if self._already_have_subtitle else sub_filenames
         return files_to_delete, info
 
+    def embed_lyrics(self, input_files):
+        audio_file = input_files[0]
+        subs = input_files[1]
+        if not subs.endswith('.lrc'):
+            raise PostProcessingError('LRC subtitles required. Use "--convert-subs lrc" to convert')
+        else:
+            with open(subs, 'r', encoding='utf-8') as f:
+                lyrics = f.read().strip()
+            if audio_file.endswith('.mp3'):
+                audio = mutagen.id3.ID3(audio_file)
+                audio.add(mutagen.id3.USLT(encoding=3, lang='eng', desc='', text=lyrics))
```

Comment:
We can't know the language of the lyrics file, so we should use `und` instead of `eng` to not mislabel it, right? `desc=''` is the default, so no need to provide it. I would also use `id3.Encoding.UTF8` to be explicit instead:
```suggestion
                audio.add(mutagen.id3.USLT(mutagen.id3.Encoding.UTF8, lang='und', text=lyrics))
```

Reply:
Use the language code of the subtitle